azafea/cli/commands.py

```diff
@@ -52,6 +54,10 @@ def register_commands(subs: argparse._SubParsersAction) -> None:
     replay.add_argument('queue', help='The name of the queue to replay, e.g "ping-1"')
     replay.set_defaults(subcommand=do_replay)
 
+    refresh = subs.add_parser('refresh-views',
```

Comment:
Did we ever work out where to trigger this? It will presumably need to run daily, soon after midnight UTC. (Are incoming requests processed in order? Do we know what the typical latency is? e.g. if it typically takes 5-10 mins for a request to hit the DB, the query should run no sooner than 00:10 UTC.)

Reply:
I'll add it to our deployment over on the `eos-administration` side. 00:30 UTC seems good enough to me.

Reply:
BTW, the PR for the deployment side is at https://github.com/endlessm/eos-administration/pull/916.